atlp-rwanda/Dev-Pulse-FN
# feature(filtering): engineers & ratings filter

#### What does this PR do?

- this PR implements filtering engineers by cohort and program on
  managers side
- you can filter trainee ratings based on the program

#### Description of Task to be completed?

- we should have functional filter that with list of cohorts and 
- functional filter with list of programs

#### How should this be manually tested?

- clone the repo `git clone {pass url here}`
- `yarn install ` to install all dependencies
- `git checkout ft-filter-engineers` to get to this branch
- `yarn dev` to start the project

Make sure you are using `nodejs version 10`

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[]()
#### Screenshots (if appropriate)

UI at managers side how it should look like

![manager ui](https://user-images.githubusercontent.com/47631065/128955519-1ca563ea-2ac6-4985-a77b-9988428a7c51.PNG)

UI on a trainee side here is how it should look like

![trainee ui](https://user-images.githubusercontent.com/47631065/128955715-8d76cd6e-ec38-4b1c-aaa3-543f193c37ee.PNG)



#### Questions:
N/A
